lib/drivers/tandemDriver.js

```diff
@@ -821,6 +821,12 @@ module.exports = function (config) {
 
   var listenForPacket = function (command,args,callback) {
 
+    var abortTimer = setTimeout(function () {
+      clearInterval(listenTimer);
```

Comment:
`retryTimer` should always fire before `abortTimer`, as `abortTimer` waits twice as long before firing.